kubernetes/run-unit-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+# We must change local_settings.py, because django search for configmap which is added in kubernetes.
+echo "from .testing import *" > "/srv/http/concent_api/concent_api/settings/local_settings.py"
```

Comment:
Why is this necessary? Youre using `--settings=concent_api.settings.testing` below and it should be enough to make `manage.py` use `testing.py`.

Reply:
I saw this option, but when I change `local_settings.py`, I'm sure that django use appriopriate settings.

Reply:
Modifying `local_settings.py` in an ad-hoc way like seems very hacky. I strongly prefer the solution with `--settings`.

Reply:
You can simply mount a volume. As long as the file is there when we run the container, it's fine.

Reply:
I resolved this problem by creating empty file in `.config_map.local_settings`